### ISSUE TITLE
Bug : The correct answer is not visually highlighted when a user selects the wrong answer in the quiz game.

### ISSUE THREAD

**Issue by aialok** [SnowScriptWinterOfCode/QuickQuiz#25]
Issue Summary:
- The correct answer is not visually highlighted when a user selects the wrong answer in the quiz game. The desired behavior is to temporarily highlight both the correct and incorrect answers for feedback. The current implementation only highlights the selected (incorrect) answer.

Proposed Solution:
- Modify the code to add the "correct" class to the correct answer choice when the user selects an incorrect answer. Remove both "correct" and "incorrect" classes after a delay for a temporary display of the correct answer.

**Comment by Nitya-Pasrija** [comment]
Congratulations! Your first ever issue with us in this repository has been created. Please wait while it gets assigned to you.

**Comment by Akanchha112** [comment]
Hello @aialok! Unfortunately, someone has already raised this issue and they are assigned to complete it. We will let you know if they complete the task or not until then this issue will be on hold.
